server/src/handlers/file_handler.rs

```diff
@@ -158,6 +158,13 @@ pub async fn get_file_handler(
     user: Option<LoggedUser>,
     _required_user: RequireAuth,
 ) -> Result<HttpResponse, actix_web::Error> {
+    let download_enabled = std::env::var("DOCUMENT_DOWNLOAD_FEATURE").unwrap_or("off".to_string());
```

Comment:
Don't we want to default to on?

Reply:
Hmmm, yeah. Let me change it. Conflicted bc Match was concerned from a security perspective

Reply:
Eh, changed my mind again. Leaving default off to reduce the odds of them messing up.